website/docs/snowflake.md

```diff
@@ -58,4 +53,45 @@ CATALOG=<catalog_name>
 METADATA_FILE_PATH='path/to/metadata/<VERSION>.metadata.json';
 ```
 
-Once the table creation succeeds you can start using the Iceberg table as any other table in Snowflake.
\ No newline at end of file
+Once the table creation succeeds you can start using the Iceberg table as any other table in Snowflake.
+
+### Method 2: Using XTable APIs to sync with Snowflake Catalog directly
+
+#### Pre-requisites:
+
+* Build Apache XTable™ (Incubating) from [source](https://github.com/apache/incubator-xtable)
+* Download `iceberg-aws-X.X.X.jar` from the [Maven repository](https://mvnrepository.com/artifact/org.apache.iceberg/iceberg-aws)
```

Comment:
[Clarification] Are AWS libraries required?

Reply:
For Snowflake, we don't need iceberg-aws, it contains integrations with glue, dynamodb etc. 
https://github.com/apache/iceberg/tree/main/aws/src/integration/java/org/apache/iceberg/aws 


> I'm not even sure what libraries would be needed for GCP and Azure

For snowflake we need permissions (IAM for AWS, service account for GCP etc.) and external volume setup. 
https://docs.snowflake.com/en/user-guide/tables-iceberg-configure-external-volume#create-an-external-volume 

XTable can already read from S3/GCS/Azure Blob/HDFS using the hadoop library dependencies. 
https://github.com/apache/incubator-xtable/blob/main/pom.xml#L360

Reply:
> For snowflake we need permissions (IAM for AWS, service account for GCP etc.) and external volume setup.

Please confirm if my understanding below is correct.
Iceberg supports various catalogs, including JDBC and REST. The Snowflake catalog appears to be JDBC-based [1]. Therefore, when connecting XTable to the Snowflake catalog and updating Iceberg tables, a Snowflake JDBC driver should be a dependency [2]. Iceberg’s JDBC catalog clients should not need Spark or AWS dependencies. However, if someone wants to follow this tutorial end-to-end, they may need Spark runtime and AWS libraries.

If this is correct, it would be helpful to separate the prereqs into two sections: one for what XTable needs and another for the tutorial prerequisites.

[1] https://www.snowflake.com/en/blog/iceberg-tables-catalog-support-available-now/
[2] https://iceberg.apache.org/docs/1.5.0/jdbc/